test/srt/openai/test_serving_chat.py

```diff
@@ -662,98 +594,18 @@ def test_response_format_json_object(self, chat_handler):
 class TestUtilityFunctions:
```

Comment:
lets remove the `test_create_error_response_functionality` and `test_create_streaming_error_response_functionality`.
I saw they are also tested in `test_serving_completions.py`. It is duplicated.

We can add them to a test_serving_base.py later.